### PR TITLE
Fix Versions Browsing Clicking on Chrome

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx [hunk 1]
```diff
@@ -590,10 +597,19 @@ const VersionsNavigator = ({
...
+                <List
+                  style={{
+                    width: "100%",
+                  }}
+                  containerStyle={{
+                    width: "100%",
+                    maxWidth: "100%",
+                  }}
+                  width={1}
+                  height={500}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:609]: Maybe we can increment this height temporarily to 800px or 1100px

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx [hunk 2]
```diff
@@ -400,24 +400,31 @@ const VersionsNavigator = ({
     setSelectedItems(cloneState);
   };
 
-  const renderVersion = (elementIndex: number) => {
-    const item = filteredRecords[elementIndex];
-    const versOrd = versions.length - versions.indexOf(item);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:405]: Please don't delete this line as it is required to display the correct version number

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx [hunk 3]
```diff
@@ -400,24 +400,31 @@ const VersionsNavigator = ({
...
+  }: ListRowProps) => {
     return (
       <FileVersionItem
+        style={style}
+        key={key}
         fileName={actualInfo?.name || ""}
-        versionInfo={item}
-        index={versOrd}
+        versionInfo={filteredRecords[index]}
+        index={index}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:416]: Version order is incorrect. Current version shouldn't be 0. 
```suggestion
        index={versOrd}
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:609]: set to `800`

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:405]: done

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx:416]: done

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by prakashsvmx — APPROVED** [review]
:+1:

**Pull request merged**